### PR TITLE
KA Cosmetic and Tracer Mods don't take up space, change name now

### PR DESCRIPTION
**Pull request by ghost**
All they do is change the icon of the KA to look nicer, they shouldn't take up 10% mod capacity.
Also made them change the name of the KA to super and hyper.

:cl:
tweak: Kinetic Accelerator Cosmetic and Tracer Modkits now don't use mod capacity. Cosmetic kits change the name of the KA.
/:cl:

### CODE REVIEW

**Comment by Emilc4h** [comment]
I could kiss you right now.

**Comment by GunHog** [comment]
Mining borgs will love you for this, as their KAs have 80% base capacity!

**Comment by jughu** [comment]
👎👎👎👎👎👎
It uses %10 to be extra baddass nostelgia wise

**Comment by Fox-McCloud** [comment]
Good change; it was annoying to have a super beefed up KA, but it looks like a stock one purely because I couldn't equip non-mechanical mods to it.

**Comment by ChangelingRain** [comment]
Should probably make tracer rounds also cost 0.

File: code/modules/projectiles/guns/energy/kinetic_accelerator.dm [hunk 1]
```diff
@@ -376,28 +376,32 @@
 /obj/item/borg/upgrade/modkit/chassis_mod
 	name = "super chassis"
 	desc = "Makes your KA yellow. All the fun of having a more powerful KA without actually having a more powerful KA."
-	cost = 10
+	cost = 0
 	denied_type = /obj/item/borg/upgrade/modkit/chassis_mod
 	var/chassis_icon = "kineticgun_u"
+	var/chassis_name = "super-kinetic acclerator"
```
Comment (Markolie) [inline code/modules/projectiles/guns/energy/kinetic_accelerator.dm]: Typo: acclerator

Comment (uraniummeltdown) [inline code/modules/projectiles/guns/energy/kinetic_accelerator.dm]: thanks 👍

**Pull request merged**